portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,11 +37,26 @@ public void init(){
         quotes.add("You can't have everything. Where would you put it?");
         quotes.add("Well-behaved women seldom make history.");
     }
+    public ArrayList messages(){
+        m = new ArrayList<>();
+        m.add("I am a message.");
+        m.add("Wow that last message is so self aware.");
+        m.add("Wait, you all can talk?");
+        return m;
+    }
+    public String toJson(ArrayList<String> alist){
+        Gson gson = new Gson();
+        String json = gson.toJson(alist);
+        return json;
+    }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String quote = quotes.get((int) (Math.random() * quotes.size()));
```

Comment:
I don't think you need the `quote` variable anymore, since this method now operates on messages derived from `message()` instead.
Similarly -- will you still be using the top-level `quotes` field and its initialization in `init()` for future functionality?

Reply:
I most likely will later, but I'm going to remove it for now.